config.json

```diff
@@ -346,6 +346,12 @@
       ]
     },
     {
+      "slug": "spiral-matrix",
+      "difficulty": 5,
```

Comment:
Interesting exercise!
Maybe difficulty 5 could be a bit too high if you consider a possible simple recursive solution with shrinking squares? (although your sample solution is of course much more interesting from a theoretical point of view).